doc/tutorials/using_windows_as_uos.rst

```diff
@@ -0,0 +1,375 @@
+.. _using_windows_as_uos:
+
+Using Windows as Guest OS on ACRN
+#################################
+This tutorial describes how to launch Windows as the Guest OS on the ACRN hypervisor.
+
+Hardware setup
+**************
+The following Intel Kaby Lake NUCs are verified:
+
+.. csv-table::
+   :header: "Platform Model", "Bios Version", "Download Link"
+
+   "NUC7i7DNHE", "DNKBLi7v.86A.0052.2018.0808.1344", "`link <https://downloadcenter.intel.com/download/28886?v=t>`__"
+   "NUC7i5DNHE", "DNKBLi5v.86A.0060.2018.1220.1536", "`link <https://downloadcenter.intel.com/download/28885?v=t>`__"
+
+ACRN Service OS Setup
+*********************
+You may refer to the steps in :ref:`getting-started-apl-nuc` for
+Intel NUC to setup ACRN on KBL NUC. After following the steps in that guide,
+you should be able to launch a Clear Linux UOS successfully.
+
+Setup for Using Windows as Guest OS
+***********************************
+All the patches to support WaaG have been upstreamed, you can download them
+from the acrn-hypervisor repository.
+
+Build ACRN EFI Images
+=====================
+   #. Follow the steps described at :ref:`getting-started-building` to set up the build environment
+   #. Use ``make`` command to compile the ``acrn.efi`` and ``acrn-dm``::
+
+      $ git clone https://github.com/projectacrn/acrn-hypervisor.git
+      $ cd acrn-hypervisor
+      $ make FIRMWARE=uefi BOARD=kbl-nuc-i7
+
+   #. You will get the outputs from::
+
+      $ build/hypervisor/acrn.efi
+      $ build/devicemodel/acrn-dm
+
+   #. Replace the ``acrn.efi`` and ``acrn-dm`` on your NUC:
+
+      a. Login ACRN Service OS and then ``mount`` EFI partition to ``/boot``
+      #. ``scp`` the ``acrn.efi`` and ``acrn-dm`` from your host::
+
+         # scp <acrn.efi from your host> /boot/EFI/acrn/
+         # scp <acrn-dm from your host> /usr/bin/
+         # chmod +x /usr/bin/acrn-dm && sync
+
+Build Service OS Kernel
+=======================
+   #. Follow the steps described at :ref:`getting-started-building` to set up the build environment
+   #. Follow the steps below to build ACRN kernel::
+
+      $ WORKDIR=`pwd`;
+      $ JOBS=`nproc`
+      $ git clone -b master https://github.com/projectacrn/acrn-kernel.git
+      $ cd acrn-kernel && mkdir -p ${WORKDIR}/{build,build-rootfs}
+      $ cp kernel_config_uefi_sos ${WORKDIR}/build/.config
+      $ make olddefconfig O=${WORKDIR}/build && make -j${JOBS} O=${WORKDIR}/build
+      $ make modules_install INSTALL_MOD_PATH=${WORKDIR}/build-rootfs O=${WORKDIR}/build -j${JOBS}
+
+Update Kernel on KBL NUC
+========================
+   #. Copy new kernel image (bzImage) and it's modules to target machine::
+
+      # scp <your host>:$WORKDIR/build/arch/x86/boot/bzImage /boot/bzImage
+      # scp -r <your host>:$WORKDIR/build-rootfs/lib/modules/* /lib/modules/
+      # cp /boot/loader/entries/acrn.conf  /boot/loader/entries/acrngt.conf
+
+   #. Modify ``acrngt.conf`` to the content as given below:
+
```

Comment:
After you left-align the numbered list, please move the code block info to the left as well:
#. Modify ``acrngt.conf`` to the content as given below:
    .. code-block:: none
       title The ACRNGT Service OS
       linux /bzImage